src/main/java/mate/academy/library/dto/category/CategoryDto.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.library.dto.category;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+
+@Data
+public class CategoryDto {
+    private Long id;

```

Comment:
Incoming DTO should not have id as it may provoke unwanted replacement instead of creation

Reply:
please address the points mentioned by previous reviewers
![image](https://github.com/BakhmetIvan/spring-boot/assets/49767049/39a1a9ed-475a-49fe-8e6c-44c7338a90ff)